static/app/components/overlay.tsx

```diff
@@ -124,7 +124,7 @@ const Overlay = styled(
           : {style};
 
       return (
-        <motion.div {...props} {...animationProps} ref={ref}>
+        <motion.div {...props} {...animationProps} data-overlay ref={ref}>
```

Comment:
Added a data attribute so we could target it with CSS - don't love it, but I couldn't find an existing attribute to target